src/main/java/page/LoginPage.java

```diff
@@ -1,32 +1,44 @@
 package page;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
 
 public class LoginPage extends BasePage {
 
-    private static final By LOGIN_BUTTON = By.linkText("Log in");
-    private static final By USERNAME_FIELD = By.id("passp-field-login");
-    private static final By PASSWORD_FIELD = By.cssSelector("[data-t='field:input-passwd']");
-    private static final By SUBMIT_BUTTON = By.cssSelector("[type='submit']");
-    private static final By USER_NAME = By.linkText("sea123alena");
+    @FindBy(linkText = "Log in")
+    private WebElement login_button;
+    @FindBy(id = "passp-field-login")
+    private WebElement username_field;
```

Comment:
use camel case instead of underscore and please separate annotation blocks with empty line. Ex.:
```

    @FindBy(linkText = "Log in")
    private WebElement loginButton;

    @FindBy(id = "passp-field-login")
    private WebElement usernameField;
```